src/core/scene/a-scene.js

```diff
@@ -320,7 +364,16 @@ module.exports = registerElement('a-scene', {
         if (this.isPlaying) {
           this.tick(time, timeDelta);
         }
-        this.effect.render(this.object3D, camera);
+        // Postproc must be explicitly enabled by a component or system.
```

Comment:
the final render to screen happens in a compositor system, the setup now is more akin to @donmccurdy physics where a proxy component controls the system. This approach is actually more proper as there should be only one place where the screen is drawn. Hence the need to have a way to explicitly toggle post proc. I also find it more convenient as otherwise you'd have to remove and readd several components to check the base render vs the post proced one